src/main/scala/scynamo/ScynamoDecoder.scala

```diff
@@ -170,17 +174,19 @@ trait DefaultScynamoDecoderInstances extends ScynamoDecoderFunctions with Scynam
 
   implicit def fieldDecoder[K, V](implicit V: Lazy[ScynamoDecoder[V]]): ScynamoDecoder[FieldType[K, V]] =
     new ScynamoDecoder[FieldType[K, V]] {
-      override def decode(attributeValue: AttributeValue) =
+      override def decode(attributeValue: AttributeValue): EitherNec[ScynamoDecodeError, FieldType[K, V]] =
         V.value.decode(attributeValue).map(field[K][V])
-      override lazy val defaultValue =
+      override lazy val defaultValue: Option[FieldType[K, V]] =
         V.value.defaultValue.map(field[K][V])
     }
 }
 
 trait ScynamoIterableDecoder extends LowestPrioAutoDecoder {
   import scynamo.syntax.attributevalue._
 
-  def iterableDecoder[A, C[x] <: Iterable[x]](implicit element: ScynamoDecoder[A], factory: Factory[A, C[A]]): ScynamoDecoder[C[A]] =
+  import scala.language.higherKinds
```

Comment:
Stray import

Reply:
IntelliJ wants me to put the feature flag there for higher-kinded types as advanced language feature. Does it hurt?

Reply:
I wonder why it doesn't complain for unused import 😄 - I would either enable the flag in `scalacOptions` or leave it alone.

Reply:
I just needed to activate the feature flag in my IntelliJ Scala compiler settings. It's gone now.